testing/internal/e2e/boundary/target.go

```diff
@@ -98,6 +98,34 @@ func CreateNewAddressTargetCli(t testing.TB, ctx context.Context, projectId stri
 	return newTargetId
 }
 
+// CreateNewSshTargetWithStorageBucketCli uses the cli to create a new SSH target using an
+// address in boundary. It enables session recording and sets a storage bucket.
+// Returns the id of the new target.
+func CreateNewSshTargetWithStorageBucketCli(t testing.TB, ctx context.Context, projectId, defaultPort, address, bucketId string) string {
```

Comment:
Trying to think about how to structure this method. We will, for a separate test, need to create an ssh target (with or without session recording). Is there a clean way to have a single method for this? Maybe we use the `opt` pattern as seen in `CreateNewTargetCli`?